sdcm/cluster.py

```diff
@@ -3846,12 +3846,15 @@ def get_grafana_screenshot_and_snapshot(self, test_start_time=None):
         if not test_start_time:
             self.log.error("No start time for test")
             return {}
-
+        date_time = datetime.utcnow().strftime('%Y%m%d_%H%M%S')
         for node in self.nodes:
             screenshot_collector = GrafanaScreenShot(name="grafana-screenshot",
                                                      test_start_time=test_start_time)
             screenshots = screenshot_collector.collect(node, self.logdir)
-            screenshots = [S3Storage().upload_file(screenshot, Setup.test_id()) for screenshot in screenshots]
+            screenshots = [S3Storage().upload_file(screenshot,
```

Comment:
you didn't follow coding standards, please read again the  "best practices" doc that you acknowledged and fix accordingly

Reply:
read and fixed.